ansible/configs/ocp-workshop/env_vars.yml

```diff
@@ -406,7 +406,7 @@ instances:
 
   - name: "node"
     count: "{{node_instance_count}}"
-    public_dns: false
```

Comment:
Is this related to this "reboot the vms" topic ? It doesn't look like it.

Reply:
Yes. It needs an EIP because the IPs change and then AAD doesn't know who to talk to any more. Now what we REALLY should be doing according to @sborenst is to run a dynamic inventory after the reboot. Sha doesn't want to do the reboot - I however strongly feel we should do it. If you can tell me how to update the inventory after the reboot I think he'd be OK with it and we wouldn't need EIPs for the nodes. Of course this is tactical until we have the Packer images. So maybe we postpone until Packer.